src/main/java/org/assertj/core/api/recursive/comparison/RecursiveComparisonDifferenceComparator.java

```diff
@@ -0,0 +1,31 @@
+package org.assertj.core.api.recursive.comparison;
+
+import org.assertj.core.internal.FieldByFieldComparator;
+
+/**
+ * Compares two objects field/property by field/property recursively using the new {@link RecursiveComparisonDifferenceCalculator} api,
+ * configured by the given {@link RecursiveComparisonConfiguration}.
+ */
+public class RecursiveComparisonDifferenceComparator extends FieldByFieldComparator {
```

Comment:
Add a `toString` method to show the `RecursiveComparisonConfiguration` used otherwise we get the `FieldByFieldComparator` one which is misleading:
```
when comparing elements using field/property by field/property comparator on all fields/properties
Comparators used:
- for elements fields (by type): {Double -> DoubleComparator[precision=1.0E-15], Float -> FloatComparator[precision=1.0E-6]}
```
^^ this is not correct as the `RecursiveComparisonConfiguration` comparators were used.